redis/commands/search/commands.py

```diff
@@ -506,7 +506,7 @@ def _get_aggregate_result(self, raw, query, has_cursor):
 
         return AggregateResult(rows, cursor, schema)
 
-    def profile(self, query, limited=False):
+    def profile(self, query, query_params=None, limited=False):
```

Comment:
This now changes the API order - which makes it breaking.  Can we instead put query_params after limited?

Reply:
We should probably also change the documentation to help people understand that it's a query + query_params.